core-bundle/src/Command/SymlinksCommand.php

```diff
@@ -173,6 +180,12 @@ private function symlinkThemes(): void
         }
     }
 
+    private function symlinkTcpdf(): void
+    {
+        $coreBundle = new \ReflectionClass($this->bundles['ContaoCoreBundle']);
```

Comment:
How does this work when you have `contao/contao` installed? There is no `ContaoCoreBundle` then, is there?

Reply:
There is? https://github.com/contao/contao/blob/master/core-bundle/src/ContaoCoreBundle.php

Reply:
I have just realized this as well. 😄